jekyll/_cci2/security-server.adoc

```diff
@@ -98,7 +98,7 @@ If you are getting started with CircleCI, there are some points you can ask your
   ** literally printing secrets in your codebase or in your shell with `echo`.
   ** programs or debugging tools that print secrets on error.
 * Consult your VCS provider's permissions for your organization (if you are in an organization) and try to follow the https://en.wikipedia.org/wiki/Principle_of_least_privilege[Principle of Least Privilege].
-* Use Restricted Contexts with teams to share environment variables with a select security group. Read through the <<contexts#restricting-a-context,contexts>> document to learn more.
+* Use Restricted Contexts with teams to share environment variables with a select security group. Read through the https://circleci.com/docs/2.0/contexts#restricting-a-context[contexts] document to learn more.
 * Ensure you audit who has access to SSH keys in your organization.
 * Ensure that your team is using Two-Factor Authentication (2FA) with your VCS (https://help.github.com/en/articles/securing-your-account-with-two-factor-authentication-2fa[Github 2FA], https://confluence.atlassian.com/bitbucket/two-step-verification-777023203.html[Bitbucket]). If a user's GitHub or Bitbucket account is compromised a nefarious actor could push code or potentially steal secrets.
-* If your project is open source and public, please make note of whether you want to share your environment variables. On CircleCI, you can change a project's settings to control whether your environment variables can pass on to _forked versions of your repo_. This is **not enabled** by default. You can read more about these settings and open source security in our <<oss#security,Open Source Projects document>>.
\ No newline at end of file
+* If your project is open source and public, please make note of whether you want to share your environment variables. On CircleCI, you can change a project's settings to control whether your environment variables can pass on to _forked versions of your repo_. This is **not enabled** by default. You can read more about these settings and open source security in our https://circleci.com/docs/2.0/oss/index.html#security[Open Source Projects Document].
```

Comment:
GH won't let me add a suggested change for some reason, but we don't need the index.html in there, then good to go. Thanks so much for finding these issues :-D